sessiondtls.go

```diff
@@ -30,9 +33,18 @@ func newSessionDTLS(connection *coapNet.Conn, srv *Server) (networkSession, erro
 		return nil, ErrInvalidBlockWiseSzx
 	}
 
+	dtlsConn := connection.Connection().(*dtls.Conn)
+	cert := dtlsConn.RemoteCertificate()
+	flatCerts := bytes.Join(cert, nil)
+	peerCertificates, err := x509.ParseCertificates(flatCerts)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Pls add the code under `if len(cert) > 0 { ... }` because we want to support DTLS with PSK

Reply:
Changed in ff8d51a. I had to move it below creation of  `sessionDTLS` in order to be able to set the `s.peerCertificates` based on whether we have parsed certificates or not.